website/docs/cloud-docs/api-docs/audit-trails.mdx

```diff
@@ -108,23 +108,327 @@ $ curl \
+| `resource.type`        | string       | Type of resource (for example, `run`)                               |
+| `resource.action`      | string       | Action audited (for example, `applied`)                             |
+| `resource.meta`        | map            | Key-value metadata about this audited event (defaults to `null`)                 |
+
+The following audit trail events _only_ contain these standard fields:
+
+| Event                      | Action                                                       | Description                                                |
+|----------------------------|--------------------------------------------------------------|------------------------------------------------------------|
+| `agent`                    | `destroy`                                                    | Logged when an agent is destroyed.                         |
+| `authentication_token`     | `create`, `show`, `destroy`                                 | Events related to authentication tokens.                  |
+| `configuration_version`    | `show`, `download`                                           | Events related to configuration versions.                 |
+| `notification_configuration` | `create`, `update`, `destroy`, `enable`                     | Events related to notification configurations.            |
+| `oauth_client`             | `create`, `update`, `destroy`                                | Events related to OAuth clients.                           |
+| `oauth_token`              | `index`, `show`, `update`, `destroy`                        | Events related to OAuth tokens.                            |
+| `organization`             | `create`, `update`, `destroy`                               | Events related to organizations.                           |
+| `organization_user`        | `create`, `update`, `destroy`                               | Events related to organization users.                      |
+| `policy`                   | `update`, `destroy`                                          | Events related to policies.                                |
+| `policy_check`             | `override`                                                   | Events related to policy checks.                           |
+| `policy_config`            | `create`                                                     | Events related to policy configurations.                  |
+| `policy_set`               | `destroy`                                                    | Events related to policy sets.                             |
+| `policy_version`           | `create`                                                     | Events related to policy versions.                         |
+| `var` (policy set parameters)           | `destroy`                                                     | Events related to destroying policy set parameters                         |
```

Comment:
I am unsure if we want to differentiate in the public documentation here about `policy set parameters` and `environment variables` or `workspace variables` I guess. I wonder if we merge them into the one `var` and denote these happen across the board for these resources.

@carolinaborim can double check, but I believe any kind of `var` will emit the exact same kind of actions.

Reply:
Yes, they emit! 
I think it makes sense to merge them!

Reply:
Sweet! I removed this row, and generalized the last `var` row accordingly:

```
| `var`                  | `index`, `show`, `create`, `update`, `destroy`              | Events related to variables.   |                         ```